src/components/button/button.spec.tsx

```diff
@@ -1,17 +1,150 @@
-import { render } from '@testing-library/react';
+import 'jest-styled-components';
+import { render, fireEvent, screen, findByText, getByRole } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import { AuthProvider } from '../../helpers';
-import { lightTheme } from '../../styles';
+import { lightTheme, darkTheme } from '../../styles';
 import { Button } from './button';
+import { pxToRem } from '../../styles';
 
 describe('Button Component', () => {
 	it('should render a default button', () => {
+
+		const { getByText, getByRole } = render(
+			<AuthProvider>
+				<Button onClick={() => console.log('test')}>Primary default Button</Button>
+			</AuthProvider>
+		);
+		expect(getByText(/Primary default Button/)).toHaveStyle(
```

Comment:
I get a typescript error for all `expect` calls => can you check how to resolve them?